examples/example-gateway/clients/quux/quux.go

```diff
@@ -21,12 +21,15 @@
 package quux
 
 import (
-	module "github.com/uber/zanzibar/examples/example-gateway/build/clients/quux/module"
+	"github.com/uber/zanzibar/examples/example-gateway/build/clients/quux/module"
+	"github.com/uber/zanzibar/examples/example-gateway/build/gen-code/clients/foo/base/base"
 )
 
 // Client is a custom client that does nothing yet
 type Client interface {
-	Echo(string) string
+	EchoString(string) string
+	//EchoMessage(m *base.Message) *base.Message
```

Comment:
Can you add a reference to tracking task for why these are commented out in another comment?

Reply:
I was experimenting, should not need to be commented out, I will update